### DISCUSSION TITLE
Github repo connecting to intellij

### DISCUSSION THREAD

**Discussion by radiobone** [community/community#73717] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How do I connect my github project directly to intellij?

**Comment by hm0223** [comment, marked as answer]
To connect your GitHub project directly to IntelliJ, follow these steps:

1. Open IntelliJ and click on "Get from Version Control" in the welcome screen or go to "VCS" > "Get from Version Control" in the main menu.
2. In the "Get from Version Control" dialog box, select "Git" as the version control system.
3. Enter the URL of your GitHub repository in the "URL" field. For example: https://github.com/username/repository-name.git
4. Choose a directory where you want to clone the repository on your local machine.
5. Click on "Clone" to start the cloning process.
6. Once the cloning is complete, IntelliJ will automatically detect that it is a GitHub project and prompt you to import the project settings from the repository.
7. Click on "Import Project" and IntelliJ will download the project files and settings from the repository.
8. After the import is complete, you can open the project in IntelliJ and start working on it.
